src/components/Header/Header.css

```diff
@@ -0,0 +1,63 @@
+header > h1 {
+  font-family: 'Caveat', cursive;
+  font-size: 3rem;
+  font-weight: 700;
+  text-align: center;
+  margin: .5rem;
+}
+
+.tree-button {
+  box-sizing: border-box;
+  background-color: #fff;
+  cursor: pointer;
+  display: flex;
+  align-self: center;
+  line-height: 1;
+  margin: 0 30px 0 30px;
+  padding: 1.1em 2.5em;
+  text-decoration: none;
+  text-align: center;
+  text-transform: uppercase;
+  font-family: 'Lato', sans-serif;
+  font-weight: 700;
+  border: 4px solid #adff91;
+  border-radius: 1rem;
+  color: #000;
+  position: relative;
+  overflow: hidden;
+  z-index: 1;
+  -webkit-transition: color 150ms ease-in-out;
+  transition: color 150ms ease-in-out;
+  box-shadow: 0px 3px 5px grey;
+}
+
+.tree-button:hover {
+  color: #fff;
+  text-shadow: 2px 2px 2px gray;
+}
+
+.tree-button:after {
+  content: '';
+  position: absolute;
+  display: block;
+  top: 0;
+  left: 50%;
+  -webkit-transform: translateX(-50%);
+  transform: translateX(-50%);
+  width: 0;
+  height: 100%;
+  background: #adff91;
+  z-index: -1;
+  -webkit-transition: width 150ms ease-in-out;
+  transition: width 150ms ease-in-out;
+}
+
```

Comment:
Since the buttons indicate which page a user is on, it might be nice if the one of the current page stays highlighted while the user is on the page. The other consideration with that is I'm not sure if there is enough contrast between the green and white colors to pass an accessibility audit.